src/controller/sql/mod.rs

```diff
@@ -557,7 +561,7 @@ impl SqlIncorporator {
         &mut self,
         q: SqlQuery,
         query_name: String,
-        is_leaf: bool,
+        _is_leaf: bool,
```

Comment:
Good catch, this is a bug. The call to `self.add_select_query()` on line 653 should use `is_leaf` instead of always passing `true`.